index.js

```diff
@@ -1,39 +1,86 @@
+      if (err) throw err;
+      var row = result[0],
+          pos = row.pos, 
+          neg = row.neg, 
+          tot = pos + neg, 
+          per = Math.floor( pos / tot * 100 ); 
+      res.send( `{ summary: '${rating(per)}', percent: ${per}, positive: ${pos}, negative: ${neg}, total: ${tot} }` )
+      // ex. { summary: 'Positive', percent: 82, positive: 4677, negative: 1218, total: 77856 }
+  })
+
 });
 
-app.get('/api/reviewcount/recent/:productId', ( req, res) => {
-  res.send( sampleRecent );
+
+app.get('/api/reviewcount/recent/:gameId', ( req, res ) => {
+
+    var sqlText = `SELECT SUM(positive) as pos, SUM(negative) as neg ` +
+                  `FROM reviews_graph WHERE date >= CURDATE()-30 AND date <= CURDATE() ` +
+                  `AND gameid = ${ req.params.gameId };`;
+  
+    db.query( sqlText, ( err, result, fields ) => {
+        if (err) throw err;
+        var row = result[0],
+            pos = row.pos, 
+            tot = pos + row.neg, 
+            per = Math.floor( pos / tot * 100 ); 
+        res.send( `{ summary: '${rating(per)}', percent: ${per}, total: ${tot} }` )
+        // ex. { summary: 'Positive', percent: 82, positive: 4677, negative: 1218, total: 77856 }
+    })
+
+  });
+
+
+app.get('/api/reviewcount/detail/:gameId', ( req, res ) => {
+
+  var sqlText = `SELECT CONCAT ( Year(date), '-', LPAD( Month(date), 2, '0'), '-01' ) as month, ` +
+                `SUM(positive) as pos, SUM(negative) as neg FROM reviews_graph ` +
+                `WHERE gameid = ${ req.params.gameId } GROUP BY month ORDER BY month;`
+
+  db.query( sqlText, (err, result, fields) => {
+      if (err) throw err;
+      res.send( `{ [ ` + result.map( row => `{ month: '${ row['month'] }', pos: ${ row['pos'] }, neg: ${ row['neg'] } }`).slice( 0, -1 ) + ` ] }`  );
+      // ex. { [ { month: '2019-10-01', positive: 4720, negative: 1591} ... ] }
+    })
+    
 });
 
-var sampleAllSummary = { summary: 'Positive', total: 77856 }
-var sampleRecentSummary = { summary: 'Very Positive', total: 246 }
 
-// converts from MySQL DATETIME: {2019-10-31 0:00:00}
+app.get('/api/reviewcount/recent/detail/:gameId', ( req, res ) => {
 
-var sampleAll = { summary: 'Positive', total: 77856, detail: [
-	{month: '2019-10-01 0:00:00', positive: 4720, negative: 1591},
-  {month: '2019-11-01 0:00:00', positive: 5980, negative: 1042} ]}
+  var sqlText = `SELECT CONCAT ( Year(date), '-', LPAD( Month(date), 2, '0'), '-', LPAD( Day(date), 2, '0') ) as day, ` +
+                `SUM(positive) as pos, SUM(negative) as neg `+
+                `FROM reviews_graph WHERE date >= CURDATE()-30 AND date <= CURDATE() ` +
+                `AND gameid = ${ req.params.gameId } GROUP BY day ORDER BY day;`
 
-var sampleRecent = { summary: 'Very Positive', total: 246, detail: [
-  {month: '2020-07-01 0:00:00', positive: 133, negative: 0},
-  {month: '2020-07-02 0:00:00', positive: 113, negative: 3} ]}  
\ No newline at end of file
+  db.query( sqlText, (err, result, fields) => {
```

Comment:
Between this line (88) and line 74, the spacings are fairly inconsistent:

`app.get('/api/reviewcount/recent/detail/:gameId', ( req, res ) => {` (open parens no space, spaces around callback) versus
`db.query( sqlText, (err, result, fields) => {` (open parens with space, no spaces around callback)

While I understand & respect your views on spacing your code, maybe it would be a good idea to set up a linter with a recommended eslint config to maintain consistency across your codebase. (HR recommends [their own](https://github.com/hackreactor-labs/eslint-config-hackreactor), of course, but it's based on Airbnb's style guide.)